src/Components/homepage.jsx

```diff
@@ -1,17 +1,46 @@
 import { Link } from "react-router-dom";
 import { useState, useEffect } from "react";
+import { getCategories } from "../utils/api";
 
 
 
-export default function Homepage (){
+export default function Homepage() {
+  const [categories, setCategories] = useState([]);
+  const [loading, setLoading] = useState(true);
 
+  useEffect(() => {
+    getCategories()
+      .then((response) => {
+        setCategories(response);
+      })
+      .then(() => {
+        setLoading(false);
+      });
+  }, []);
 
+  if (loading) {
+    return <p>Page is currently loading</p>;
+  }
 
-    return(
-        <>
-        Homepage WIP
-        <br/>
-        Select one of the categories below to get started
-        </>
-    )
-}
\ No newline at end of file
+  return (
+    <>
+      <br />
+      Select one of the categories below to get started
+      <ul>
+        {categories.map((category, index) => {
+          return (
+                <button key = {index}>
+            <Link to={`/reviews?category=${category.slug}`}>
+              <li>
+                  Category: {category.slug}
+                  <br />
+                  Description:{category.description}
+              </li>
+            </Link>
+                </button>
+          );
+        })}
+      </ul>
```

Comment:
This is all fine, but from a user experience there's just quite alot of text. It might be worth just having the category name on it's own? or maybe adding some formatting/styling to maybe have the category name as the main text, and the description in smaller text separated from the category name title... just to make it easier to read. Entirely up to you how you do this, but I think it could be slightly more user friendly to read either way.